homeassistant/components/camera/icons.json

```
@@ -0,0 +1,19 @@
+{
+  "entity_component": {
+    "_": {
+      "default": "mdi:video",
+      "state": {
+        "off": "mdi:video-off"
+      }
+    }
+  },
+  "services": {
+    "disable_motion_detection": "mdi:motion-sensor-off",
+    "enable_motion_detection": "mdi:motion-sensor",
+    "play_stream": "mdi:play",
+    "record": "mdi:record-rec",
+    "snapshot": "mdi:camera",
+    "turn_off": "mdi:video",
+    "turn_on": "mdi:video-off"
```

The off state is `video-off`. Shouldn't the turn_off service represent the icon of the desired state? 
```suggestion
    "turn_off": "mdi:video-off",
    "turn_on": "mdi:video"
```

---

👍 nice catch